index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Landing Page</title>
+    <link rel="stylesheet" href="css/style.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+    <div class="portfolio-body">
+        <h1>
+            My portfolio
+        </h1>
+        <header id="portfolio">
+            <span>Filter: </span>
+            <button class="portfolio-button--active">All</button>
+            <button class="portfolio-button">
+                <i class="far fa-gem"></i>
+                Design
+            </button>
+            <button class="portfolio-button">
+                <i class="far fa-image"></i>
+                Photos
+            </button>
+            <button class="portfolio-button">
+                <i class="fas fa-paint-brush"></i>
+                Art
+            </button>
+        </header>
+        <div class="cards-wrapper">
+            <div class="triple-cards">
```

Comment:
hm, I didn't expect to see containers as `triple-cards`. I expected `cards-wrapper` to be a flex-box with different amount of cards in a line for different screens.
Maybe I'm wrong, but changing `float: left` to `display: flex` for different viewports looks a little odd.
Be careful with `float`. I'd use flex-box in many cases